src/main/java/maksym/fedorenko/bookstore/controller/BookController.java

```diff
@@ -29,7 +34,25 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
     public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
+
+    @PutMapping("/{id}")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestDto bookDto) {
```

Comment:
just FYI

you may also use `ResponseEntity` if you find that beneficial

it would look like:
```
return ResponseEntity.accepted().build();
```